### DISCUSSION TITLE
Why i cant install wire-elements modal v-2.0 in my v3livewire "Solved"

### DISCUSSION THREAD

**Discussion by abdouchamkha** [wire-elements/modal#351] [General]
![image](https://github.com/wire-elements/modal/assets/75368032/bf3004e2-07fe-4dad-9408-e381c92d886d)

**Comment by PhiloNL** [comment]
Problem 1:
- Add `"wire-elements/modal": "^2.0@beta"` manually to your `composer.json` instead of using the CLI command (windows doesn't handle the `^` symbol very well.
- Try setting your composer.json minimum stability to `dev` or `beta`.

Problem 2:
- `wire-elements/pro` requires a license.

**Reply by abdouchamkha** [reply]
thanks for your fast replay no its work's for me 
with `composer require wire-elements/modal:^2.0.5@beta`
and its show in composer.json like that "wire-elements/modal": "2.0.5-beta",

**Reply by PhiloNL** [reply]
Great
